op-node/flags/flags.go

```diff
@@ -193,17 +187,24 @@ var (
 		Value:    time.Second * 3,
 		Category: L1RPCCategory,
 	}
-	L1BlobRPCRateLimit = &cli.Float64Flag{
-		Name:     "l1.blob-rpc-rate-limit",
-		Usage:    "Optional self-imposed global rate-limit on L1 blob RPC requests, specified in requests / second. Disabled if set to 0.",
-		EnvVars:  prefixEnvVars("L1_BLOB_RPC_RATE_LIMIT"),
+	L1ArchiveBlobRpcAddr = &cli.StringFlag{
+		Name:     "l1.archive-blob-rpc",
+		Usage:    "Optional address of L1 archive blob endpoint to use. Multiple alternative addresses are supported, separated by commas, and will rotate when error",
+		Required: false,
+		EnvVars:  prefixEnvVars("L1_ARCHIVE_BLOB_RPC"),
+		Category: RollupCategory,
+	}
+	L1ArchiveBlobRpcRateLimit = &cli.Float64Flag{
```

Comment:
I think there is no need to add `Archive` for parameter `blob-rpc-rate-limit` and `blob-rpc-max-batch-size` since they also apply to non-archive urls(the l1 part).